### DISCUSSION TITLE
possible to make sequence mode one-way?

### DISCUSSION THREAD

**Discussion by nancyxyzou** [openseadragon/openseadragon#2499] [Q&A] [answered]
Is there a way to make sequence mode only allow navigation in one direction? Is there perhaps a way to disable the back button so that you can only move forward in the sequence of images?

**Comment by iangilman** [comment, marked as answer]
Interesting! No, we don't have a feature for that. Sequence mode is pretty simple, though... You could easily write your own version of it. 

Looking at the code, you might also be able to get away with something like: 

```
viewer.removeControl(viewer.previousButton.element);
delete viewer.previousButton;
```

**Reply by nancyxyzou** [reply]
Thank you so much!
